.github/workflows/elixir.yml

```diff
@@ -34,44 +50,51 @@ jobs:
     - name: Restore deps cache
       uses: actions/cache@v2
       with:
-        path: deps
-        key: deps-${{ matrix.otp }}-${{ matrix.elixir }}-${{ hashFiles('**/mix.lock') }}
+        path: |
+          deps
+          _build
```

Comment:
I'd rather not combine these, `deps` changes infrequently, whereas `_build` will change with every single code change.

Reply:
`deps` contains the source code of dependencies. `_build` contains the compiled `.beam` files for these dependencies as well as for your project. And the project compiles files are usually just the small part of `_build` folder content. So there is no actual reason to store these files separately - updating files in `deps` will lead to updating the `_build` folder.

Reply:
But code updates in absinthe itself (which is far more frequent) change `_build` which would re-cache the `deps` w/o needing too

Reply:
Yes, but saving cache takes less than 2 seconds even for such a large project like absinthe.